NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -1,13 +1,32 @@
-from NuRadioReco.utilities import units, fft, ice, geometryUtilities
+from NuRadioReco.utilities import units, ice, geometryUtilities
 from NuRadioReco.modules.base.module import register_run
 import NuRadioReco.framework.channel
+import NuRadioReco.framework.sim_station
 import NuRadioReco.detector.antennapattern
 import logging
 import numpy as np
-import matplotlib.pyplot as plt
 import scipy.constants
 import scipy.interpolate
-import pygdsm
+
+logger = logging.getLogger('NuRadioReco.channelGalacticNoiseAdder')
+
+try:
+    from radiocalibrationtoolkit import *  # Documentation: https://github.com/F-Tomas/radiocalibrationtoolkit/tree/main contains SSM, GMOSS, ULSA
```

Comment:
Can we add this as an optional dependency in the installation script? Also can we use explicit imports here?

Reply:
The problem is that some of the modules included in the radiocalibrationtoolkit are not straightforward to install in my experience.
https://github.com/F-Tomas/radiocalibrationtoolkit/tree/main the documentation states to import the packages in the toolkit like this...

Reply:
@karenterveer I also tried to install the radiocalibrationtoolkit, but it seems it's a bit picky (e.g. it requires an older version of pygdsm, there's also an install-time dependency on `wheel` that's not documented because it still relies on an outdated `setup.py` to install)... if you still remember how to install these modules it would be nice if you could document the instructions!

Another reason why we should get this right is that right now this module is incompatible with pygdsm and radiocalibrationtoolkit at the same time - we need pygdsm 1.5 to avoid an ImportError for `GlobalSkyModel16`, but radiocalibrationtoolkit still expects `GlobalSkyModel2016` which was apparently deprecated at some point in pygdsm.